src/Spectre.libGenetic/Individual.h

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 
 #pragma once
 #include <vector>
+#include "DataTypes.h"
```

Comment:
Missing directory prefix. It won't work through NuGet package.

Reply:
done